src/Persistence_matrix/include/gudhi/Persistence_matrix/columns/naive_vector_column.h

```diff
@@ -224,7 +224,13 @@ class Naive_vector_column : public Master_matrix::Row_access_option,
 };
 
 template <class Master_matrix>
-inline Naive_vector_column<Master_matrix>::Naive_vector_column(Column_settings* colSettings)
+using STD_naive_vector_column = Naive_vector_column<Master_matrix, std::vector<typename Master_matrix::Matrix_entry*> >;
```

Comment:
What do you think of Naive_std_vector_column and Naive_small_vector_column, to keep the qualifier of vector next to the word vector?

Reply:
Sounds much better! Changed it.